GMCPagingScrollView/GMCPagingScrollView.m

```diff
@@ -152,31 +152,31 @@ - (CGRect)frameForPageAtActualIndex:(NSUInteger)index {
 }
 
 - (void)layoutSubviews {
-	[super layoutSubviews];
-	
-	CGRect frameForScrollView = [self frameForScrollView];
-	if (!CGRectEqualToRect(self.scrollView.frame, frameForScrollView)) {
+    self.inLayoutSubviews = YES;
```

Comment:
Did you check going from the nav to a page with index > 50, for example? It might not take you to the right page is my concern. If it works, let me know and I'll merge.